.travis.yml

```diff
@@ -5,17 +5,27 @@ language: generic
 
 matrix:
   include:
-    - env: PYTHON=python2 PYTHON_VER=  PYTAVE=no SYMPY_VER=1.0 OCT_PPA=yes DOCTEST=yes COLUMNS=80
-    - env: PYTHON=python3 PYTHON_VER=3 PYTAVE=no SYMPY_VER=1.0 OCT_PPA=yes DOCTEST=yes COLUMNS=80
+    - env: PYTHON=python2 PYTHON_VER=  PYTAVE=no SYMPY_VER=1.0 OCT=ppa   DOCTEST=yes COLUMNS=80
+    - env: PYTHON=python3 PYTHON_VER=3 PYTAVE=no SYMPY_VER=1.0 OCT=ppa   DOCTEST=yes COLUMNS=80
+    - env: PYTHON=python2 PYTHON_VER=  PYTAVE=no SYMPY_VER=1.0 OCT=daily DOCTEST=yes COLUMNS=80
+    - env: PYTHON=python3 PYTHON_VER=3 PYTAVE=no SYMPY_VER=1.0 OCT=daily DOCTEST=yes COLUMNS=80
+  allow_failures:
+    - env: PYTHON=python2 PYTHON_VER=  PYTAVE=no SYMPY_VER=1.0 OCT=daily DOCTEST=yes COLUMNS=80
+    - env: PYTHON=python3 PYTHON_VER=3 PYTAVE=no SYMPY_VER=1.0 OCT=daily DOCTEST=yes COLUMNS=80
 
 # need octave devel pkgs for doctest (has compiled code as of July 2015)
 before_install:
-  - if [ "x$OCT_PPA" = "xyes" ]; then
+  - if [ "x$OCT" = "xppa" ]; then
         sudo apt-add-repository -y ppa:octave/stable;
     fi
   - sudo apt-get update -qq
   - sudo apt-get install -y liboctave-dev
   - sudo apt-get install -y octave
```

Comment:
I think it's nicer to keep them for now, because otherwise we would have to list all the individual shared libraries and dependencies that Octave needs to work (as I have to do [here](https://github.com/mtmiller/octave-snapshot/blob/master/Dockerfile)).